lua/super-commit/super-commit.lua

```diff
@@ -18,11 +19,9 @@ function M.set_autocmd()
     -- pattern = 'COMMIT_EDITMSG',
     callback = function()
       window_open.setup()
-      local bufnum_table = window_open.bufnum_table
-      local commands = git_cmds.commands
-      for k, cmd in pairs(commands) do
-          git_cmds.show_command_output(bufnum_table[k], cmd)
-      end
+      git_cmds.show_command_output_v2(win_manage_table["git-status"])
+      git_cmds.show_command_output_v2(win_manage_table["filelist"])
+      git_cmds.show_command_output_v2(win_manage_table["git-diff-init"])
```

Comment:
Here `win_manage_table` is contained in the `git_cmds` module. So you only need to pass keys for these functions:
```suggestion
      git_cmds.show_command_output_v2("git-status")
      git_cmds.show_command_output_v2("filelist")
      git_cmds.show_command_output_v2("git-diff-init")
```

Reply:
Thank you for your suggestion.　I think it is better.
We will make the tables easier to access in the future.